### PR TITLE
Support Promremotebench Metrics && Add Monitoring Dash

### PR DESCRIPTION
**Pull request by notbdu**
**What this PR does / why we need it**:

Adds prom deployment to promremotebench server and adds a monitoring dash for both m3db node and promremotebench.
Also increases prom retention to 48h.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note
NONE
```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note
NONE
```

### CODE REVIEW

File: scripts/vagrant/provision/manifests/kube-prometheus/grafana-dashboardDefinitions.yaml [hunk 1]
```diff
@@ -943,6 +943,6534 @@ items:
   metadata:
     name: grafana-dashboard-k8s-cluster-rsrc-use
     namespace: monitoring
+- apiVersion: v1
+  data:
+    m3db_node_details.json: |-
+      {
```
Comment (arnikola) [inline scripts/vagrant/provision/manifests/kube-prometheus/grafana-dashboardDefinitions.yaml]: Is there a way we can pull this out into a separate json representation of the dash and include that here instead?

Comment (notbdu) [reply scripts/vagrant/provision/manifests/kube-prometheus/grafana-dashboardDefinitions.yaml]: I don't think so, I couldn't find anything. It looks like this is the way to do it if we write out the yamls ourselves. 

Although it seems like using `kubectl` you can create config maps from files with a flag.

Comment (arnikola) [reply scripts/vagrant/provision/manifests/kube-prometheus/grafana-dashboardDefinitions.yaml]: Maybe even just as part of a build step that will swap out a template var for the dash json or something?

Comment (notbdu) [reply scripts/vagrant/provision/manifests/kube-prometheus/grafana-dashboardDefinitions.yaml]: We could prob use perl or sed but I think it's too much for these test dash defs. 

All other dash are also defined in the same way in the file.

Comment (arnikola) [reply scripts/vagrant/provision/manifests/kube-prometheus/grafana-dashboardDefinitions.yaml]: kk

**Pull request merged**